io/include/detray/io/json/json_algebra_io.hpp

```diff
@@ -15,12 +15,12 @@
 /// JSON I/O extension
 namespace detray {
 
-void to_json(nlohmann::ordered_json& j, const transform_payload& t) {
+inline void to_json(nlohmann::ordered_json& j, const transform_payload& t) {
```

Comment:
Just curious: Did this cause any errors?

Reply:
multiple definition error in `traccc/test` directory.
This didn't happen in `traccc/examples`

Reply:
Interesting, thanks

Reply:
No i couldnt find such mistake in include. There was another error as well whic i have never seen before

Reply:
Yeah so multiple def of n file varaiables and all json functions without inline type